coinbase_commerce/error.py

```diff
@@ -99,23 +103,42 @@ class WebhookInvalidPayload(APIError):
     pass
 
 
-def build_api_error(response, blob=None):
+def _build_api_error(code, body=None, headers=None, blob=None,
+                     json_loads=json.loads):
```

Comment:
Is there ever any time somebody would want to specify the json_loads function? I don't think so, unless it is being mocked